src/App.js

```diff
@@ -1,25 +1,30 @@
 import React from "react";
-import logo from "./logo.svg";
 import "./App.css";
 import styled from "styled-components";
 import { Switch, Route } from "react-router";
 import LandingPage from "./views/LandingPage";
 import { useSelector, useDispatch } from "react-redux";
 import StyledButton from "./components/StyledButton";
 import { fetchUser } from "./actions";
+import { createGlobalStyle } from 'styled-components';
+import reset from 'styled-reset';
 
 function App( props ){
   
   const user = useSelector( state => state.usersReducer );
   const dispatch = useDispatch();
+
+  const GlobalStyle = createGlobalStyle`
+    ${reset}
+  `
   
   const handleButtonClick = () => {
     fetchUser( dispatch );
   };
   
   return (
-    
     <StyledApp className="App">
+      <GlobalStyle />
```

Comment:
I think a better place for this would be in the index.js

Reply:
Makes sense.